_html/templates/dimond-sponsors-page.php

```diff
@@ -0,0 +1,8 @@
+<?php
+
+include "inc/header.inc.php";
+
+include "modules/dimond-sponsor-content.php";
```

Comment:
You have a typo here. Please rename everything that has dimond into diamond. Both in SCSS and PHP files. Thanks :).